k8s/deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: opensergo-control-plane
+  namespace: opensergo-system
+  labels:
+    app: opensergo-control-plane
+spec:
+  selector:
+    matchLabels:
+      app: opensergo-control-plane
```

Comment:
Good idea. This can be improved with Helm chart version :)

Reply:
PS: we'll also need to test with multiple control plane instances.

Reply:
> PS: we'll also need to test with multiple control plane instances.

multiple instances may have questions in data version synchronize

Reply:
> > PS: we'll also need to test with multiple control plane instances.
> 
> multiple instances may have questions in data version synchronize

This may depend on whether the control plane is stateless or not.

Reply:
yeah, need a mid-storage to store the state-datas